Simplenote/Int+Extensions.swift

```diff
@@ -0,0 +1,7 @@
+import Foundation
+
+extension Int {
+    func roundToNearestFive() -> Int {
+        return 5 * Int((Double(self) / 5.0).rounded())
+    }
```

Comment:
Since this API is *not* generic (it's very, very specific to just one use case), perhaps it could be better off as a function in `FontSettings` ? (or elsewhere?)